### ISSUE TITLE
unexpected free argument `aur`

### ISSUE THREAD

**Issue by alanvardy** [fosskers/cargo-aur#25]
I am currently unable to run `cargo aur`, and don't know how to interpret the error message.

```
 > cargo aur
/home/vardy/.cargo/bin/cargo-aur: unexpected free argument `aur`
```

Any idea what I am doing wrong? How can I investigate this further?

**Comment by fosskers** [comment]
I'm able to reproduce. Looking into it now.

**Comment by fosskers** [comment]
Note that this doesn't occur when just running `cargo run` from the `cargo-aur` repo itself (which is a common pattern for me while developing).

**Comment by fosskers** [comment]
Fix released as `1.7.1`. Thanks for reporting!

**Comment by alanvardy** [comment]
Thank you for the speedy response! Confirmed fixed on my end :+1:
